deltachat-ios/Controller/ChatListController.swift

```diff
@@ -169,6 +166,12 @@ class ChatListController: UITableViewController {
         updateTitle()
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
```

Comment:
all tableViewControllers and UITableViewCells can react on font size changes with the help of `override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?)`